src/Options/index.js

```diff
@@ -241,6 +242,10 @@ export interface ParseServerOptions {
   playgroundPath: ?string;
   /* Callback when server has started */
   serverStartComplete: ?(error: ?Error) => void;
+  /* Callback when server has started and before running schemas migration operations if schemas key provided */
+  beforeSchemasMigration: ?() => void | Promise<void>;
```

Comment:
It seems the validations are missing for the new parameters, see for example `validateSecurityOptions`:

https://github.com/parse-community/parse-server/blob/9ea355b4635226ae4da17c8cc5fb0321e3fdec5e/src/Config.js#L116-L130